src/profile-logic/active-tab.js

```diff
@@ -61,6 +61,8 @@ export function computeActiveTabTracks(
   state: State
 ): ActiveTabTimeline {
   // Global tracks that are certainly global tracks.
+  // FIXME: We should revert back to full view if we failed to find a track
+  // index for the main track.
   const mainTrackIndexes = [];
```

Comment:
Question: What happens here if an empty array is used?

Reply:
If the array is empty, we should abort and revert to full view because there is nothing to show. It's not handled currently, and there is this fixme to remind me :) I have a WIP patch for that as well, probably I'll open a PR for that tomorrow.